Formula/v2raya.rb

```diff
@@ -0,0 +1,35 @@
+class V2raya < Formula
+  desc "A web GUI client of Project V, which supports V2Ray, Xray, SS, SSR, Trojan and Pingtunnel."
+  homepage "https://v2raya.org"
+  url "https://github.com/v2rayA/v2rayA/archive/refs/tags/v1.5.7.zip"
+  sha256 "1a635783559fd4ef29fa99b2300d9f76c1d9c7638b12851288199a12e1676af5"
+  license "AGPL-3.0-Only"
+  head "https://github.com/v2rayA/v2rayA.git", branch: "master"
+
+  depends_on "go" => :build
+  depends_on "node@16" => :build
+  depends_on "yarn" => :build
+  depends_on "v2ray"
+
+  livecheck do
+    url :stable
+    strategy :github_latest
+  end
+
+  def install
+    ENV.deparallelize
+    system "bash", "-c", "export OUTPUT_DIR=../service/server/router/web; cd ./gui; yarn; yarn update; yarn build --outDir ../service/server/router/web/"
```

Comment:
This should be things like `ENV["VAR"] = "a"`, not a bash version of homebrew features.